stackblitz-starter/README.md

```diff
@@ -0,0 +1,3 @@
+# Fluent UI StackBlitz Starter
```

Comment:
This probably isn't the best folder for this template but it keeps it out of the way of existing things for now.

Reply:
I don't think it's a good idea to just "throw" random folders around repo from maintenance and guaranteing that "things work". 

How urgent is this in general ?

Reply:
Sorry I wasn't clearer. I'm not suggesting this template ultimately lives here, rather that I put it here to keep it out of the way while we discuss its merits.

Reply:
right I understood what you meant, but in order to proceed with this there are couple of things to resolve:

- this "starter" is not something we want to officially ship as it's falls short on any kind of configuration as codesandbox did before
- it will be confusing to ship 2 official "starters" later on as we will developed and provide proper vite template
- this code is generated https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-storybook-addon-export-to-sandbox/src/sandbox-scaffold.ts#L32, the best solution would be to have 1 source of truth
- who would be the main consumer of this "non reliable" starter template ?